docs/src/design/stack/crypto_ops.md

```diff
@@ -141,7 +141,7 @@ At the high-level, the operation does the following:
 - Using $x$ and $\alpha$, folds the query values $q_0, ..., q_3$ into a single value $r$.
 - Compares the previously folded value $pe$ to the appropriate value of $q_0, ..., q_3$ to verify that the folding of the previous layer was done correctly.
 - Computes the new value of $poe$ as $poe' = poe^4$ (this is done in two steps to keep the constraint degree low).
-- Increments the layer address pointer by $2$.
+- Increments the layer address pointer by $8$.
```

Comment:
We increment by 8 because we fold by 4 and each element is an element in the quadratic extension field - right?

Reply:
It's because at `layer_ptr` we are storing the FRI layer commitment and at  `layer_ptr + 4` we are storing  `[a0, a1, t_depth, d_size]` where `a = (a_0, a_1)` is the FRI folding challenge and `t_depth`, `d_size` are the tree and domain size for the current layer.
The folded values are the leaves of the tree and are provided ND